domains/ananya-solanki.is-a.dev.json

```
@@ -0,0 +1,4 @@
+{
+"owner": { "username": "ananyasolanki1", "email": "[ananya.solanki10@gmail.com](mailto:ananya.solanki10@gmail.com)" },
```

invalid email address
```suggestion
"owner": { "username": "ananyasolanki1", "email": "ananya.solanki10@gmail.com" },
```